src/home_robot/home_robot/mapping/voxel/feature/utils.py

```diff
@@ -0,0 +1,124 @@
+import os
+import struct
+from copy import deepcopy
+from typing import List
+
+import numpy as np
+import pytorch3d
+import torch
+from PIL import Image
+
+COLOR_LIST = [
+    [0.0, 0.0, 1.0],
+    [0.0, 1.0, 0.0],
+    [1.0, 0.0, 0.0],
+    [0.0, 1.0, 1.0],
+    [1.0, 0.0, 1.0],
+    [1.0, 1.0, 0.0],
+    [0.0, 0.5, 0.0],
+    [0.5, 0.0, 0.0],
+    [0.0, 0.5, 0.5],
+    [0.5, 0.0, 0.5],
+    [0.5, 0.5, 0.0],
+    [0.0, 0.0, 0.5],
+    [0.0, 0.5, 1.0],
+]
+
+
+def write_pointcloud(filename, xyz_points, rgb_points=None):
```

Comment:
type hints?

Reply:
unused right now -- not sure if we want to keep or not